src/components/ReportActionItemIOUQuote.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import _ from 'underscore';
+import styles from '../styles/styles';
+import ReportActionPropTypes from '../pages/home/report/ReportActionPropTypes';
+import RenderHTML from './RenderHTML';
+
+const propTypes = {
+    // All the data of the action
+    action: PropTypes.shape(ReportActionPropTypes).isRequired,
+};
+
+const ReportActionItemIOUQuote = ({action}) => (
+    <View style={[styles.chatItemMessage]}>
+        {_.map(_.compact(action.message), (fragment, index) => {
+            const viewDetails = '<br /><a href="#">View Details</a>';
+            const html = `<blockquote>${fragment.text}${viewDetails}</blockquote>`;
```

Comment:
This was left simple to keep making progress with IOU. I'll be replacing it with an actual component in the IOU Details issue so that I can link to a modal when clicking 'View Details'

Reply:
Ok. My concern is that someone else might see this and think it's correct, but if there are plans to fix it quickly maybe that's fine.

Reply:
So the styling of this doesn't quite match our mockups:

![image](https://user-images.githubusercontent.com/2319350/114237367-d920f980-9937-11eb-97bf-0a48d7f14d61.png)


versus

![image](https://user-images.githubusercontent.com/2319350/114237429-f1911400-9937-11eb-8507-5a3ec0913d25.png)

Is this the PR where I should share my feedback on how to accomplish the mockups, or will this be tackled elsewhere?

Reply:
Hi @shawnborton 

I didn’t update the styles of HTML tags since the plan is to replace HTML part with standalone components later.

Reply:
Hey @shawnborton. 

Let's keep that feedback for the new PR which will replace the HTML. I'll make sure to tag you in the PR.